### ISSUE TITLE
Upgrade for Mautic 4.0

### ISSUE THREAD

**Issue by OttoSte99** [nickian/mautic-whitelabeler#107]
Hello,
considering that **Mautic 4** will be released on May 24 (https://forum.mautic.org/t/getting-ready-for-mautic-4-all-details-timelines/18692) I'm expecting that a new **mautic-whitelabeler** version will be released soon.
In the meantime, I'm asking if the installation of the newest plugin version will keep the current customization settings. Can you confirm me that?
Thank you in advance.

**Comment by nickian** [comment]
The whitelabeler will not be updated until I can review the final release code for Mautic 4. I don't currently support development versions of Mautic for white labeling.

Each release of the whitelabler includes support for the newest version and past versions. As long as Mautic 4 isn't set up completely different than past versions, this should still be the case. If it's entirely different, I'll have to change the code a bit and release a new version only for 4.0 and up.

Every time you update Mautic, your whitelabel settings will get overwritten by the default Mautic styles. I don't recommend updating production instances of Mautic until a new release has been thoroughly tested and the bugs are worked out. Plus, the whitelabeler probably will not be ready the day of the new release, so you would be stuck with default styles for a little while.

Keep in mind this is simply a free script that I created for personal use that I decided to share with the world. It's not a paid or commercial project, so support is very limited. Basically, use at your own discretion and there are no guarantees on new version releases or times for those releases.
